pkgs/applications/misc/khal/default.nix

```diff
@@ -86,6 +86,5 @@ python3.pkgs.buildPythonApplication rec {
     homepage = "http://lostpackets.de/khal/";
     license = licenses.mit;
     maintainers = with maintainers; [ gebner ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ gebner ];
    platforms = platforms.unix;
```

Reply:
Since we are using buildPythonApplication, the platforms should be set implicitly.

Reply:
Oh, my bad.